price-feeder/config/config_test.go

```diff
@@ -199,7 +201,8 @@ global-labels = [["chain-id", "umee-local-testnet"]]
 	_, err = tmpFile.Write(content)
 	require.NoError(t, err)
 
-	cfg, err := config.ParseConfig(tmpFile.Name())
+	logger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr}).Level(zerolog.InfoLevel).With().Timestamp().Logger()
```

Comment:
let's move this logger to a variable at the top of the file so we don't have to keep making new ones